packages/contracts/contracts/TellerV2.sol

```diff
@@ -720,7 +739,9 @@ contract TellerV2 is
         Bid storage bid = bids[_bidId];
         bid.state = BidState.CLOSED;
 
-        collateralManager.lenderClaimCollateral(_bidId);
+        //collateralManager.lenderClaimCollateral(_bidId);
+
+        _getCollateralManagerForBid(_bidId).lenderClaimCollateral(_bidId);
```

Comment:
I think we should emit the `LoanLiquidated` event here as well from now on. granted it won't back track for previous liquidations but it should have been here to be begin with and would simplify the subgraph since I'm currently doing some hacky stuff to update the status of a bid when the collateral is claimed and the current status of the bid isn't `REPAID`

Reply:
define the word This please